notification/notification-sdk/src/main/java/org/sunbird/notification/beans/Constants.java

```diff
@@ -9,4 +9,5 @@ public class Constants {
   public static final String EMAIL_SERVER_USERNAME = "sunbird_mail_server_username";
   public static final String FROM_EMAIL = "fromEmail";
   public static final String EMAIL_ORG_IMG_URL = "orgImageUrl";
+  public static final String Is_TLS_Enable = "sunbird_mail_server_use_tls";
```

Comment:
Rename the below variables
Is_TLS_Enable --> EMAIL_SERVER_TLS_ENABLED
sunbird_mail_server_use_tls -> mail_server_use_tls

Reply:
Done